opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -102,6 +104,38 @@ def get_context(self) -> 'SpanContext':
             A :class:`.SpanContext` with a copy of this span's immutable state.
         """
 
+    def set_attribute(self: 'Span',
```

Comment:
I don't see the type annotation used for self parameters anywhere in the file.

Reply:
They do that in the subclass in:
opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py
Not sure if the inconsistency is on purpose